test/Driver/driver-compile.swift

```diff
@@ -113,6 +113,8 @@
 // FILELIST: -primary-file {{.*/(driver-compile.swift|empty.swift)}}
 // FILELIST: -output-filelist {{[^-]}}
 
+// UPDATE-CODE-NOT: -filelist
+// UPDATE-CODE-NOT: -output-filelist
```

Comment:
These don't actually test that it doesn't appear anywhere in the output. You'll need to do a second FileCheck run for that.